File: Core Java/inheritance-app/src/com/tecklabs/boy/Boy.java
package com.tecklabs.boy;

import com.techlabs.men.Men;

public class Boy extends Men {

	public Boy(){
		System.out.println("Boy is created....");
	}
	public void eat() {
		System.out.println("This is Eat Method of Boy class....");
	}

	@Override
	public void play() {
		System.out.println("This is play method of Boy class....");
	}
}

File: DOTNET/C#/InheritanceApp/InheritanceApp/Boy.cs
using System;


namespace InheritanceApp
{
    class Boy : Men
    {
        public Boy() {
            Console.WriteLine("Boy Class Created.......");
        }

        public void eat() {
            Console.WriteLine("This is Eat Method of Boy Class......");
        }

        public override void play() {
            Console.WriteLine("This is Play Method of Boy Class......");
        }
    }
}
